aws/logs_monitoring/release.sh

```diff
@@ -26,60 +22,101 @@ if [ -z "$2" ]; then
+        echo "Must use a version greater than the current ($CURRENT_VERSION)"
+        exit 1
+    fi
+    # Make sure we are on master
+    BRANCH=$(git rev-parse --abbrev-ref HEAD)
+    if [ $BRANCH != "master" ]; then
+        echo "Not on master, aborting"
+        exit 1
+    fi
+
+    # Confirm to proceed
+    read -p "About to bump the version from ${CURRENT_VERSION} to ${VERSION}, create a release aws-dd-forwarder-${VERSION} on Github and upload the template.yaml to s3://${BUCKET}/aws/forwarder/${VERSION}.yaml. Continue (y/n)?" CONT
+    if [ "$CONT" != "y" ]; then
+        echo "Exiting"
+        exit 1
+    fi
 
-# Bump version number
-echo "Bumping the current version number to the desired"
-perl -pi -e "s/DD_FORWARDER_VERSION = \"${CURRENT_VERSION}/DD_FORWARDER_VERSION = \"${VERSION}/g" lambda_function.py
-perl -pi -e "s/Version: ${CURRENT_VERSION}/Version: ${VERSION}/g" template.yaml
+    # Get the latest code
+    git pull origin master
 
-# Commit version number changes to git
-git add lambda_function.py template.yaml README.md
-git commit -m "Bump version from ${CURRENT_VERSION} to ${VERSION}"
-git push origin master
+    # Bump version number
+    echo "Bumping the current version number to the desired"
+    perl -pi -e "s/DD_FORWARDER_VERSION = \"${CURRENT_VERSION}/DD_FORWARDER_VERSION = \"${VERSION}/g" lambda_function.py
+    perl -pi -e "s/Version: ${CURRENT_VERSION}/Version: ${VERSION}/g" template.yaml
 
-# Create a github release
-echo "Release aws-dd-forwarder-${VERSION} to github"
-go get github.com/github/hub
-rm -f aws-dd-forwarder-*.zip
-zip -r aws-dd-forwarder-${VERSION}.zip .
-hub release create -a aws-dd-forwarder-${VERSION}.zip -m "aws-dd-forwarder-${VERSION}" aws-dd-forwarder-${VERSION}
+    # Commit version number changes to git
+    git add lambda_function.py template.yaml README.md
+    git commit -m "Bump version from ${CURRENT_VERSION} to ${VERSION}"
+    git push origin master
+
+    # Create a github release
+    echo "Release aws-dd-forwarder-${VERSION} to github"
+    go get github.com/github/hub
+    rm -f aws-dd-forwarder-*.zip
+    zip -r aws-dd-forwarder-${VERSION}.zip .
+    hub release create -a aws-dd-forwarder-${VERSION}.zip -m "aws-dd-forwarder-${VERSION}" aws-dd-forwarder-${VERSION}
+    TEMPLATE_URL="https://${BUCKET}.s3.amazonaws.com/aws/forwarder/latest.yaml"
+    FORWARDER_SOURCE_URL="https://github.com/DataDog/datadog-serverless-functions/releases/download/aws-dd-forwarder-${VERSION}/aws-dd-forwarder-${VERSION}.zip'"
+else
+    echo "About to release non-public staging version of forwarder, upload aws-dd-forwarder-${VERSION} to s3, and upload the template.yaml to s3://${BUCKET}/aws/forwarder-staging/${VERSION}.yaml"
+    # Upload to s3 instead of github
+    rm -f aws-dd-forwarder-*.zip
+    zip -r aws-dd-forwarder-${VERSION}.zip .
+    aws s3 cp aws-dd-forwarder-${VERSION}.zip s3://${BUCKET}/aws/forwarder-staging-zip/aws-dd-forwarder-${VERSION}.zip
```

Comment:
Just want to make sure we update the internal wiki with the new staging testing steps.

Reply:
Will do when this and the other PR is merged